docs/content/guides/recipes/Terraform/howto-private-registry/index.md

```diff
@@ -19,32 +19,31 @@ Before you get started, you'll need to make sure you have the following tools an
 - [rad CLI]({{< ref "installation#step-1-install-the-rad-cli" >}})
 - [Radius Bicep VSCode extension]({{< ref "installation#step-2-install-the-vs-code-extension" >}})
 - [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+- A [personal access token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens#about-personal-access-tokens) for your Git repository.
```

Comment:
This shouldn't be specific to just GitHub, as Azure DevOps, GitLab, and other git platforms are also supported. I'd make this Step 1, with links to all the supported platforms.